src/datreant/tests/test_treants.py

```diff
@@ -413,8 +425,7 @@ def test_add_wrong_keys(self, treant, key, val):
             with pytest.raises(TypeError):
                 treant.categories.add(key, val)
 
-        @pytest.mark.parametrize('key, val', [['bark', ['shaggy']],
```

Comment:
?

Reply:
This was testing that setting `list` as a value would not work. The only thing remaining currently is a faulty dictionary `{'yes'}`, I guess one could remove the parametrize decorator. Or should `datreant` actually fail, if a user tries to set a category value to an empty list>